docs/source/api/v4/servers.rst

```diff
@@ -159,6 +164,8 @@ Response Structure
 :updPending: A boolean value which, if ``true``, indicates that the server has updates of some kind pending, typically to be acted upon by Traffic Control Cache Config (:term:`t3c`, formerly ORT)
 :xmppId:     A system-generated UUID used to generate a server hashId for use in Traffic Router's consistent hashing algorithm. This value is set when a server is created and cannot be changed afterwards.
 :xmppPasswd: The password used in XMPP communications with the server
+:asns:       The :abbr:`ASN (Autonomous System Number)` associated with the cachegroups of the current server.
+	.. versionadded:: 4.1
```

Comment:
Indentation isn't sufficient to put this "under" the field list item. Right now, it's rendering as the literal text ".. versionadded:: 4.1". So in HTML:
```html
<dd class="field-even"><p>The <abbr title="Autonomous System Number">ASN</abbr> associated with the cachegroups of the current server.
.. versionadded:: 4.1</p>
</dd>
```
To get it to render properly, add a blank line between the directive and the field with which it's associated.